ci-operator/step-registry/dra/dra-example-driver/install/dra-dra-example-driver-install-commands.sh

```
@@ -0,0 +1,136 @@
+echo "Waiting for ResourceSlices to be published..."
+timeout 5m bash -c '
+while true; do
+  RESOURCE_SLICES=$(oc get resourceslice -o name 2>/dev/null | wc -l)
+  if [ "${RESOURCE_SLICES}" -gt 0 ]; then
+    echo "Found ${RESOURCE_SLICES} ResourceSlice(s)"
+    break
+  fi
+  echo "Waiting for ResourceSlices..."
+  sleep 10
+done
+'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)